packages/tools/kadena-cli/src/account/commands/accountAddManual.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk';
+import type { Command } from 'commander';
+import debug from 'debug';
+import path from 'path';
+import { defaultAccountPath } from '../../constants/account.js';
+import { updateAccountDetailsPrompt } from '../../prompts/account.js';
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+import { sanitizeFilename } from '../../utils/helpers.js';
+import { getUpdatedConfig } from '../utils/addHelpers.js';
+import { validateAccountDetails } from '../utils/validateAccountDetails.js';
+import { writeConfigInFile } from '../utils/writeConfigInFile.js';
+
+export const addAccountManualCommand: (
+  program: Command,
+  version: string,
+) => void = createCommand(
+  'add-manual',
+  'Add an existing account to the CLI',
+  [
+    globalOptions.accountAlias(),
+    globalOptions.accountName(),
+    globalOptions.fungible(),
+    globalOptions.network(),
+    globalOptions.chainId(),
+    globalOptions.publicKeys(),
+    globalOptions.predicate(),
+  ],
+
+  async function addAccount(config): Promise<void> {
```

Comment:
I think this method is still doing too much, please add a test that does add-manual and add-wallet and see how easy it is to re-use all the logic.